web/app/view/map/BaseMap.js

```diff
@@ -187,6 +187,27 @@ Ext.define('Traccar.view.map.BaseMap', {
 
         this.map.addControl(new ol.control.LayerSwitcher());
 
+        popup = new ol.Overlay.Popup();
+        this.map.addOverlay(popup);
+
+        geocoder = new Geocoder('nominatim', {
+            provider: 'bing',
+            key: server.get('bingKey'),
```

Comment:
This doesn't make sense. Why are you using bind key for nominatim?

Reply:
Changed to using OSM